pysat/instruments/pysat_testing_xarray.py

```diff
@@ -0,0 +1,165 @@
+# -*- coding: utf-8 -*-
+"""
+Produces fake instrument data for testing.
+"""
+
+import pandas as pds
+import numpy as np
+import pysat
+import xarray
+
+# pysat required parameters
+platform = 'pysat'
+name = 'testing_xarray'
+# dictionary of data 'tags' and corresponding description
+tags = {'':'Regular testing data set'}
+# dictionary of satellite IDs, list of corresponding tags
+sat_ids = {'':['']}
+test_dates = {'':{'':pysat.datetime(2009,1,1)}}
+pandas_format = False
+
+
+def init(self):
+    self.new_thing=True        
+                
+def load(fnames, tag=None, sat_id=None, sim_multi_file_right=False,
+        sim_multi_file_left=False):
+
+    # create an artifical satellite data set
+    parts = fnames[0].split('/')
```

Comment:
This should use the os.path file splitting routines for windows compatibility

Reply:
You are correct. I also switched from a date format with / to one with - to avoid the whole windows mess.